tasks/create-advisory/README.md

```diff
@@ -16,6 +18,10 @@ the ReleasePlanAdmission and Application from the Snapshot are also used. The ad
 | synchronously            | Whether the task should wait for InternalRequests to complete                             | Yes      | true                        |
 | pipelineRunUid           | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                             |
 
+## Changes since 3.1.0
+- This task now detects which secret to use for creating advisories based on the targeted quay repository.
+- Only all `redhat-pending` or all `redhat-prod` repositories may be specified in `.data.mapping`
```

Comment:
We use redhat-pending in e2e so this should not require an e2e change https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/release/pipelines/rh_advisories.go#L237